### DISCUSSION TITLE
Batch update capabilities to transition existing records to new resourceTypeGeneral values

### DISCUSSION THREAD

**Discussion by codycooperross** [datacite/datacite-suggestions#103] [General Suggestions]
### What is the problem that your suggestion solves?

A member would like to update all of their DOI records from one resourceTypeGeneral value to another. For example, from "PhysicalObject" to "Text." 

### What solution might meet your needs?

The ability to batch load global updates to DOI records.

### What alternatives have you tried or considered?

Using the REST API to update DOI records. 

### Is there anything else you would like to share?

The member only expects to use this feature annually. 

[Link for DataCite staff](https://docs.google.com/spreadsheets/d/16jQnuW_DMgbb-74Y5TByUG4IkHdvJSZ0Fe_qxQ4Xl6w/edit?gid=65276345#gid=65276345&range=106:106)

### What group(s) would benefit from your suggestion?

- [X] DataCite members registering DOIs and metadata and administering accounts
- [ ] Developers building integrations with DataCite services
- [X] Harvesters who index DataCite DOIs and metadata
- [x] Individuals or institutions analyzing DataCite DOIs and metadata
- [ ] DataCite staff
- [ ] Other

**Comment by KellyStathis** [comment]
I like this idea! To expand on the problem statement, we are hoping to improve the accuracy of resourceTypeGeneral values and make it easier for repositories to make these updates. I also checked off "Individuals or institutions analyzing DataCite DOIs and metadata" as groups who would benefit here.
